specs/Standards/Tokens/FungibleTokenMetadata.md

```diff
@@ -53,23 +63,34 @@ A fungible token contract implementing the metadata standard shall contain a fie
 
 ```ts
 type FungibleTokenMetadata = {
-    spec: string;
-    name: string;
-    symbol: string;
     reference: string;
     reference_hash: string;
-    decimals: number;
+    decimals?: number;
+    icon?: string;
+    name?: string;
+    spec?: string;
```

Comment:
`spec` should be required here. It indicates how you parse the remaining fields of the metadata object.

Reply:
the current version has it as required